### PR TITLE
[KR] Translation of week05/05.md

### PR DESCRIPTION
**Pull request by seungjaeryanlee**

### CODE REVIEW

File: docs/ko/week05/05.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+## Lecture part A
+
+We begin by introducing Gradient Descent. We discuss the intuition and also talk about how step sizes play an important role in reaching the solution. Then we move on to SGD and its performance in comparison to Full Batch GD. Finally we talk about Momentum Updates, specifically the two update rules, the intuition behind momentum and its effect on covergence.
+ -->
+
+## 이론 part A
+
+We begin by introducing Gradient Descent. We discuss the intuition and also talk about how step sizes play an important role in reaching the solution. Then we move on to SGD and its performance in comparison to Full Batch GD. Finally we talk about Momentum Updates, specifically the two update rules, the intuition behind momentum and its effect on covergence.
+
+우리는 경사하강법<sup>Gradient Descent (GD)</sup>을 소개하는 것으로 시작한다. 우리는 경사하강법을 직관적으로 이해하기 대해 토론하고, 해결책에 도달하는 데 학습률<sup>step size</sup>이 어떤 중요한 역할을 하는지에 대해서도 이야기한다. 그 후 확률적 경사하강법<sup>Stochastic Gradient Descent (SGD)</sup>와 이것의 성능을 확인하고 배치 GD와 비교한다. 마지막으로 모멘텀 업데이트<sup>momentum update</sup>에 대해 얘기할텐데, 특히 두 가지 업데이트 방식과, 모멘텀 뒤의 직관, 그리고 수렴에 미치는 영향에 대해 이야기한다.
```
[the remark this answers is not in the document]
Comment (marofmar) [reply docs/ko/week05/05.md:17]: - 이것의 성능을 확인하고 배치 GD와 비교한다. -> 이것의 성능을 확인하고 풀 배치<sup>full batch</sup> GD<sup>Gradient Descent</sup>와 비교한다.

[the remark this answers is not in the document]
Comment (seungjaeryanlee) [reply docs/ko/week05/05.md]: 리뷰 감사합니다. 첫번째는 고쳤고, 두번째는 GD 는 줄임말이라서 [한국어 번역 위키에 적힌 대로](https://github.com/Atcold/pytorch-Deep-Learning/wiki/Korean-translation#rules) 유지하는게 나을 것 같아서 놔두었습니다. 어떻게 생각하시나요?
